coding/cpp_style_guide.rst

```diff
@@ -788,25 +788,19 @@ Keeps inputs together.
 
 .. _style-guide-cpp-3-34:
 
-3-34. Uncertainty values associated with a variable SHOULD be suffixed by one of ``Var``, ``Cov``, ``Sigma``.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There is no universal suffix for uncertainties; i.e. no ``Err`` suffix will be used.
-The cases that we have identified, and their appropriate suffixes, are:
+3-34. Uncertainty values associated with a variable SHOULD be suffixed by one of ``Err`` or``Cov``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Standard deviation: ``Sigma`` (not ``Rms``, as ``rms`` doesn't imply that the mean's subtracted)
-- Covariance: ``Cov``
-- Variance: ``Var``
+Following the DPDD, we use ``Err`` to specify error quantities (as standard errors), and ``Sigma`` to specify values inherent width of a distribution or function.
 
 .. code-block:: cpp
 
-   float xAstrom;          // x position computed by a centroiding algorithm
-   float xAstromSigma;     // Uncertainty of xAstrom
-   float yAstrom;
-   float yAstromSigma;
-   float xyAstromCov;
+   float xCentroid;        // x position computed by a centroiding algorithm
+   float xCentroidErr;     // Uncertainty of xAstrom
+   float yCentroid;
+   float yCentroidErr;
+   float xyCentroidCov;    // Covariance of x/y centroid
```

Comment:
maybe include a sigma example? Otherwise the last sentence looks a bit out of place.